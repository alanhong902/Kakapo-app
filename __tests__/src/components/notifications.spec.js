import React from 'react';
import renderer from 'react-test-renderer';
import Notifications from 'components/ui/notifications/notifications';

function setup(props = {}) {
  return {
    props,
    tree: renderer.create(<Notifications {...props} />).toJSON(),
  };
}

test('<Notifications/> render', () => {
  const { tree } = setup({ notifications: {} });
  expect(tree).toMatchSnapshot();
});

test('<Notifications/> className', () => {
  const { tree } = setup({ notifications: { a: 'test' } });
  expect(tree).toMatchSnapshot();
});

test('<Notifications/> list order', () => {
  const { tree } = setup({ notifications: { a: 'test', b: 'test2' } });
  expect(tree).toMatchSnapshot();
});
