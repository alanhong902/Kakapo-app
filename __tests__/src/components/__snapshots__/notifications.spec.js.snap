// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notifications/> className 1`] = `
<aside
  className="notify notify-visible"
>
  <div>
    test
  </div>
</aside>
`;

exports[`<Notifications/> list order 1`] = `
<aside
  className="notify notify-visible"
>
  <div>
    test
  </div>
  <div>
    test2
  </div>
</aside>
`;

exports[`<Notifications/> render 1`] = `
<aside
  className="notify"
/>
`;
